src/SdfEntityCreator.cc

```diff
@@ -750,19 +755,17 @@ Entity SdfEntityCreator::CreateEntities(const sdf::Visual *_visual)
   }
 
   // store the plugin in a component
-  if (_visual->Element())
+  if (!_visual->Plugins().empty())
   {
-    sdf::ElementPtr pluginElem =  _visual->Element()->FindElement("plugin");
-    if (pluginElem)
-    {
-      this->dataPtr->ecm->CreateComponent(visualEntity,
-          components::VisualPlugin(pluginElem));
-    }
+    this->dataPtr->ecm->CreateComponent(visualEntity,
```

Comment:
Here too we could break users who are counting on the `VisualPlugin` component, I think we should create both components here

Reply:
Done in [afd30fb](https://github.com/gazebosim/gz-sim/pull/1352/commits/afd30fbb79c7c81cf9d7bb056179fa6cd92a0d94)